README.md

```diff
@@ -242,6 +228,14 @@ Run the web app container using:
 
     docker run --net='container:max-im2txt' -it webapp
```

Comment:
Same comment applies here, changing `max-im2txt` to `max-image-caption-generator`